export/main.ts

```diff
@@ -0,0 +1,89 @@
+import { PhylumApi } from 'phylum';
+import { parse } from "https://deno.land/std@0.156.0/flags/mod.ts";
+import { MultiProgressBar } from "https://deno.land/x/progress@v1.4.4/mod.ts";
+
+
+/**
+ * Fetch the project data from the Phylum API.
+ */
+async function fetchProjectData(projectId: string, group?: string): Promise<any> {
+    try {
+        const url = group ? `groups/${group}/projects/${projectId}` : `data/projects/${projectId}`;
+        const response = await PhylumApi.fetch("v0/", url, {});
+
+        if(!response.ok) {
+            console.error(`Failed to fetch project data, received HTTP error: ${response.status}`);
```

Comment:
```suggestion
            console.error(`\nFailed to fetch project data, received HTTP error: ${response.status}`);
```
This is b/c the progress bar does not write the newline and this message would show at the end of the progress bar with no spaces.